ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/create_table.md

```diff
@@ -159,6 +159,28 @@ WITH (
 );
 ```
 
+Creating a table with enabled [TTL](../../../../concepts/ttl.md) for automatically removing outdated records from the table. A record is deleted when the value of one of the columns (configurable by the user) of type: `Date`, `Datetime`, or `Timestamp` becomes greater than the current time (Epoch time). The lifespan of records can be extended (globally for the entire table) by the value specified in the `Interval` function:
+```sql
+CREATE TABLE my_table (
+    id Uint64,
+    title Utf8,
+    expire_at Timestamp
+    PRIMARY KEY (id)
+)
+WITH (
+    TTL = Interval("PT0S") ON expire_at
```

Comment:
According to https://ydb.tech/docs/en/concepts/ttl, there's also an option to use an integer column and specify in which time unit to interpret values:

> The value in the TTL column with a numeric type (Uint32, Uint64, or DyNumber) is interpreted as a [Unix time](https://en.wikipedia.org/wiki/Unix_time) value. The following units are supported (set in the TTL settings):
> 
> - Seconds.
> - Milliseconds.
> - Microseconds.
> - Nanoseconds.

It would be helpful to cover this case, too.